lisp/mastodon-tl.el

```diff
@@ -146,6 +155,71 @@ Return value from boosted content if available."
   (or (cdr (assoc field (cdr (assoc 'reblog toot))))
       (cdr (assoc field toot))))
 
+(defun mastodon-tl--relative-time-details (timestamp &optional current-time)
+  "Returns cons of (descriptive string . next change) for the TIMESTAMP.
```

Comment:
checkdoc suggests using the imperative return rather than returns. However, it looks like their are endemic issues with the docs in this file, so perhaps we can go through and fix all the doc issues in a separate PR.